windows-active-directory-mixin/README.md

```diff
@@ -0,0 +1,159 @@
+# Windows Active Directory mixin
+
+The Windows Active Directory mixin is a set of configurable Grafana dashboards and alerts.
+
+The Windows Active Directory mixin contains the following dashboards:
+
```

Comment:
nitpicky: but do we normally have a space here but not before the alerts?

Reply:
I dont think it matters as the spacing changes depending on the mixin you are looking at, but i updated it to be more inline with the style you are thinking of i believe.